test/unit/org/apache/cassandra/cql3/ast/AssignmentOperator.java

```diff
@@ -109,4 +109,14 @@ public AbstractType<?> type()
     {
         return right.type();
     }
+
+    @Override
+    public Expression visit(Visitor v)
+    {
+        var u = v.visit(this);
```

Comment:
the `visit` pattern is the same for each type, so documenting here as this is the first to show up in the diff

1) the visitor could replace the full object, this is checked by calling the visitor directly `v.visit(this)`
2) when working with sub elements we don't know if the element has sub-elements, so need to call `e.visit(v)` so it can recursively do its work.